combat-trainer.lic

```diff
@@ -1449,6 +1463,16 @@ class SpellProcess
     game_state.casting_weapon_buff = $weapon_buffs.include?(name)
     if data['ritual']
       cast_ritual(data, game_state)
+    elsif data['runestone_name']
+      if DRCI.inside?(data['runestone_name'], @settings.runestone_storage)
```

Comment:
💡 I suggest moving this into a local var like the other settings, see line 1000ish.